packages/asc-core/src/components/ButtonStyle/ButtonStyle.ts

```diff
@@ -25,7 +25,7 @@ const ButtonStyle = styled(ButtonBaseStyle)<Props>`
   & svg {
     width: 30px;
     height: 30px;
-    ${({ color, theme }) => fillSvgFromTheme(theme, color)};
+    ${({ color, theme }) => svgFill(color)({ theme })};
```

Comment:
plz find a clever way to curry theme to svgFill

Reply:
will be done in the second iteration